README.md

```diff
@@ -1,2 +1,72 @@
+
+
+## Terminology
+
+* Individual - housing insecure person using services to interact with organizations.
+* Organization - housing agencies, landlords, government agencies that will be requesting information from individuals.
+* Pod/Wallet - decentralized data/document storage built with Solid API. Individuals and organizations will have certain functinality within their pods.
+* Folder/Container - referring to organization of document data into ttl files using Solid API with Inrupt libarary.
+
+
+## User Flows
+
+1. Individual
+    * Upload and Maintain Documents
+    * Able to share documents
+    * Can copy verified documents into pod from organization(s)
+2. Organization
+    * Can request and review documents
+    * Can create verified documents
+    * Able to share verified documents with individuals
+    * Might require several users
+
+## If you're new to the team, check out our [onboarding](https://github.com/codeforpdx/PASS/blob/db/onboarding/onboarding.md) file.
+
+## Tech Overview
+
+Currently building out use-cases in vanilla javascript to determine Solid and Inrupt functionality. Once we have proof of concept, we will build out the PASS application in React.
+
+### Solid Data Pods
+
+Pods are like secure personal web servers for data.
+
+#### General User Flow of Solid Pods:
+* User registers for a WebID (kind of like an email address)
+* User data remains in pod (which is associated to WebID)
+* User authorizes applications to read/write/control data in pod
+
+#### To become familiar with how pods work:
+* Go to [Kevin's Solid Experiment](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid) on GitHub.
+* Clone repository and follow the [instructions](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid/01-one-server).
+* Feel free to play around with the pod and check out the second use case [here](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid/02-different-file-browser-application).
+
+### Inrupt JavaSctipt Client Libraries
+
+### RDF
+
+### Document Hashing
+
+
+## The Team
+
+|Back-End Dev Team    | Front-End Dev Team    | UX/UI Team | Project Management Team |
+| ------------------- | --------------------- | -----------| ------------------------|
+| Team Lead - Kevin M.| Team Lead - Jared K.  | Gabby      | Flo                     |
+| Colton              | Andy                  | Kyle       | Danica                  |
```

Comment:
I think Colton has stepped back for now.

Reply:
Should we maybe just keep you, Jared, Gabby, Kyle, Flo, me, Wilf and Katherine listed? Maybe add people after they've been on the project for a certain amount of time? Or have people add themselves as a task when they are first onboarding?

Reply:
> Should we maybe just keep you, Jared, Gabby, Kyle, Flo, me, Wilf and Katherine listed?

That sounds good to me!